src/Slices/Resource/Data/getUrl.ts

```diff
@@ -21,5 +22,9 @@ export function getUrl({
         )}`
       : "";
   const sortParam = sort ? `&sort=${sort.name}.${sort.order}` : "";
-  return `/api/v2/resource?deploy_summary=True&limit=${pageSize.value}${filterParam}${sortParam}`;
+  const currentPageParams =
```

Comment:
I believe this is something worth extracting into your handler, and exported. That way, if the logic needs to be adjusted a bit, you'll only need to change it in one place, instead of all 50 files where this is being used ;)